src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -385,6 +385,11 @@ void Thread::Destroy()
     ASSERT(m_pGCFrameRegistrations == NULL);
 }
 
+gc_alloc_context* Thread::GetAllocContext()
```

Comment:
Should it be in `thread.inl` ? It seems important that this method gets inlined.